### ISSUE TITLE
Debian Package

### ISSUE THREAD

**Issue by ssgelm** [github/git-lfs#296]
I have created a deb package for git-lfs.  Would you be interested in my submitting a PR to add the debian directory to the package so that a debian package for git-lfs can be built out of the box?  I understand that it ends up being a single binary but it's convenient for us to have a deb to easily deploy to machines.

**Comment by technoweenie** [comment]
Sure, file a PR and we can talk about it.

**Comment by technoweenie** [comment]
I just merged #299 which added build instructions for CentOS, while updating the Ubuntu instructions.  There are a couple things I noticed though:

1. It's possible to install a Go 1.0 package, when Git LFS requires Go 1.3x.
2. Ruby is required for the ronn package to build the man pages.  Can these be generated when the package is generated, so that ruby isn't required for users installing it?

Can this be solved by a proper Debian package?
